src/main/scala/tech/cryptonomic/conseil/tezos/ApiOperations.scala

```diff
@@ -85,79 +163,101 @@ object ApiOperations {
-                  Tables.Operations#TableElementType),
-                Seq]) extends JoinedTables
-
-  case class BlocksOperationGroupsAccounts
-  (table: Query[((Tables.Blocks, Tables.OperationGroups), Tables.Accounts),
-                ((Tables.Blocks#TableElementType, Tables.OperationGroups#TableElementType),
-                  Tables.Accounts#TableElementType),
-                Seq]) extends JoinedTables
-
-  case class BlocksOperationGroups
-  (table: Query[(Tables.Blocks, Tables.OperationGroups),
-                (Tables.Blocks#TableElementType, Tables.OperationGroups#TableElementType),
-                Seq]) extends JoinedTables
-
-  case class BlocksOperationsAccounts
-  (table: Query[((Tables.Blocks, Tables.Operations), Tables.Accounts),
-                ((Tables.Blocks#TableElementType, Tables.Operations#TableElementType),
-                  Tables.Accounts#TableElementType),
-                Seq]) extends JoinedTables
-
-  case class BlocksOperations
-  (table: Query[(Tables.Blocks, Tables.Operations),
-                (Tables.Blocks#TableElementType, Tables.Operations#TableElementType),
-                Seq]) extends JoinedTables
-
-  case class BlocksAccounts
-  (table: Query[(Tables.Blocks, Tables.Accounts),
-                (Tables.Blocks#TableElementType, Tables.Accounts#TableElementType),
-                Seq]) extends JoinedTables
-
-  case class Blocks
-  (table: Query[Tables.Blocks, Tables.Blocks#TableElementType, Seq]) extends JoinedTables
-
-  case class OperationGroupsOperationsAccounts
-  (table: Query[((Tables.OperationGroups, Tables.Operations), Tables.Accounts),
-                ((Tables.OperationGroups#TableElementType, Tables.Operations#TableElementType),
-                  Tables.Accounts#TableElementType),
-                Seq]) extends JoinedTables
-
-  case class OperationGroupsOperations
-  (table: Query[(Tables.OperationGroups, Tables.Operations),
-                (Tables.OperationGroups#TableElementType, Tables.Operations#TableElementType),
-                Seq]) extends JoinedTables
-
-  case class OperationGroupsAccounts
-  (table: Query[(Tables.OperationGroups, Tables.Accounts),
-                (Tables.OperationGroups#TableElementType, Tables.Accounts#TableElementType),
-                Seq]) extends JoinedTables
-
-  case class OperationGroups
-  (table: Query[Tables.OperationGroups, Tables.OperationGroups#TableElementType, Seq]) extends JoinedTables
-
-  case class OperationsAccounts
-  (table: Query[(Tables.Operations, Tables.Accounts),
-                (Tables.Operations#TableElementType, Tables.Accounts#TableElementType),
-                Seq]) extends JoinedTables
-
-  case class Operations
-  (table: Query[Tables.Operations, Tables.Operations#TableElementType, Seq]) extends JoinedTables
-
-  case class Accounts
-  (table: Query[Tables.Accounts, Tables.Accounts#TableElementType, Seq]) extends JoinedTables
+  case class BlocksOperationGroupsOperationsAccounts(
```

Comment:
I'm crying at how much better this is. Now we change schemas without fear!

Reply:
Great then!